src/js/services/auth.js

```diff
@@ -108,17 +108,18 @@ export default App.extend({
   },
   initBootstrap() {
     const d = $.Deferred();
-    $.when(Radio.request('entities', 'fetch:clinicians:current')).then(currentUser => {
+    const fetchCurrentUser = Radio.request('entities', 'fetch:clinicians:current');
+    const fetchGroups = Radio.request('entities', 'fetch:groups:collection');
+    const fetchRoles = Radio.request('entities', 'fetch:roles:collection');
+    const fetchStates = Radio.request('entities', 'fetch:states:collection');
+    $.when(fetchCurrentUser, fetchRoles, fetchStates, fetchGroups).done((currentUser, roles, states) => {
```

Comment:
👀 does this mean we can get rid of compound documents?!

Reply:
requests that can be made in parallel are far different than requests the must be made serially.   This is a trivial request.  Making an additional and separate patient request for each action that is returned on an action list is quite a bit more complicated